templates/Kibana.logrotate.j2

```diff
@@ -0,0 +1,22 @@
+{{ kibana_log_dir }}/kibana.stdout
```

Comment:
casing on the filename - I can't remember if Ansible is picky about it, but would be nice to keep everything lower case and consistent

Reply:
that comment wasn't very clear. It's the casing on the template filename I was referring to

Reply:
Makes sense. Renamed the file.